Add Tomoscan review page. (#80)
tomochain/docs: docs/tomoscan/Tomoscan.md

-   The Search button: you can search an adress in particular and check the TOMO balance, the TOMO USD value, the transactions and a 0x code.

### Transactions.
You can find here all the transactions made on Tomochain,
You can also check the Signs transactions, Pending Transactions and Other Transactions,
-   Signs transactions are transactions that are done.
-   Pending transactions are transactions that are in progress.
-   Other Transactions correspond to the Sign transactions and the Pending transactions.

### Accounts
-   Accounts includes Masternodes and Verified  Contracts,
-   Verified Contracts are contracts that were verified by the author.

### Tokens
The Tokens tab includes all the tokens that are using Tomochain and all the Token Transfers from an adress to another.

### Blocks

You can check all the blocks that were mined on Tomochain using the Search button. 

![alternative textuelle](/whole block tomoscan.jpg)

The informations available are the following :
-   Height : the number of the block.
-   Age : the time where the block was mined.
-   Txn : number of transactions made on this block.

> -   Miner : the adress of the miner that verified the block.

Author: wow, my bad for that one.

-   GasUsed: the nergy needed to mine the block.

> -   Finality : it is the guarantee that past transactions can never change. If it is higher, the security is better.

Reviewer: ```suggestion
- Finality: The percentage of the network who validated this block. When it reach 75%, the block reach it's finality state and is added permanently to the chain.
```

> If you click on a block, you will find more informations such as more detailled transactions made on the block and the Blocksigner. Blocksigner is the Masternode that created the block.

Reviewer: You can make that part more detailed

![alternative textuelle](/assets/block detailled tomoscan.jpg)

### Sign and Register

> If you register, you will be able to keep a watchlist of some adressesses, keep them on a list and receive a notification when by email when a transaction is made with this adress,

Reviewer: ```suggestion
If you register, you will be able to keep a watchlist of some addresses your interested into.
You will receive an email notification when this address perform actions.
```

> Please keep in mind that Tomoscan is subject to go through some changes. That is why your feedback is appreciated!

Reviewer: ```suggestion
Please keep in mind that TomoScan is subject to go through some changes. That is why your feedback is appreciated!
```
We always capitalise the second word like: Tomo**C**hain, Tomo**S**can, Tomo**W**allet
